app/views/email_templates/new.html.slim

```diff
@@ -0,0 +1,21 @@
+javascript:
+  $( document ).on('turbolinks:render, turbolinks:load', function() {
+    $('#email_template_kind').change(function(){
+      var probezeit = "Liebe/r $[Anrede] $[Name]\n\nWir bitten um Ihre $[Feedback_Link] für $[Begleitung_Titel].\n\nDanke im Voraus!"
```

Comment:
I've just discovered, that for our template variables it's probably the best to use `"string to %{replace} in"`.

This is, beacause:

```ruby
"here is %{replace} for me" % { replace: 'text i want to be replaced' }
```

so it will be by far the smoothest and easiest if we did it that way.